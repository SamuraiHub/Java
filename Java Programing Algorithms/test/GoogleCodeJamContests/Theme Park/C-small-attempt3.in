50
4 6 4
1 4 2 1
100 10 1
1
5 5 10
2 4 2 3 4 2 1 2 1 3
1 1 1
1
100 2 2
1 2
1000 100 10
4 3 9 1 2 5 6 9 1 10
1 10 10
4 3 4 1 2 5 6 9 1 10
1 10 9
1 1 1 1 1 1 1 1 1
1000 10 9
1 1 1 1 1 1 1 1 1
934 9 10
8 8 8 8 8 8 8 8 8 8
734 3 10
1 2 3 3 2 1 1 2 3 1
345 19 10
10 10 10 10 10 10 10 10 10 10
765 10 10
10 1 10 1 10 1 10 1 10 1
1000 100 10
10 10 10 10 10 10 10 10 10 10
734 20 10
1 2 3 4 5 6 7 8 9 10
631 16 10
10 9 8 7 6 5 4 3 2 1
4 12 8
9 6 4 6 6 6 10 7
3 8 9
6 5 4 2 3 5 3 4 5
3 11 5
1 7 5 6 3
3 7 5
4 1 7 7 7
869 7 9
6 3 3 4 6 4 5 4 4
692 14 9
6 7 10 8 5 4 3 8 2
626 12 9
8 4 10 4 5 3 7 10 9
216 11 9
1 4 7 7 6 7 3 3 5
9 29 9
10 4 6 1 2 6 10 9 9
8 22 9
1 1 6 9 7 5 5 7 8
8 24 9
10 10 8 4 4 7 4 3 1
9 25 10
5 5 3 5 5 7 1 6 9 6
15 16 9
7 8 6 5 9 6 6 8 8
151 23 10
9 6 8 5 6 7 9 7 6 5
51 22 8
6 7 6 9 8 7 6 8
221 23 9
9 6 8 9 7 7 5 8 5
331 54 10
8 8 5 6 9 6 8 6 5 7
219 11 8
7 7 9 5 7 9 6 8
97 65 10
6 7 5 6 8 9 9 8 5 9
604 26 9
9 7 5 6 7 6 7 6 5
273 13 10
7 9 5 6 9 7 8 6 7 9
148 30 6
5 9 5 7 5 5
507 45 5
5 6 6 9 9
687 83 7
5 9 6 7 7 7 9
751 12 9
7 8 8 8 5 9 5 9 6
64 97 5
8 8 7 7 7
404 16 9
6 6 5 9 8 7 7 5 5
959 77 7
8 9 7 5 5 9 5
277 18 8
6 6 6 5 5 8 9 5
340 93 6
9 6 7 6 8 7
962 4 8
4 4 4 4 4 4 4 4
248 35 8
5 7 9 6 7 9 7 8
4 75 6
9 5 5 7 6 9
220 22 9
6 7 5 6 8 6 5 8 8
