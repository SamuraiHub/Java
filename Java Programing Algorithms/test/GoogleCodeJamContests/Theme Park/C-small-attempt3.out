Case #1: 21
Case #2: 100
Case #3: 20
Case #4: 1
Case #5: 150
Case #6: 50000
Case #7: 7
Case #8: 9
Case #9: 9000
Case #10: 7472
Case #11: 1994
Case #12: 3450
Case #13: 4212
Case #14: 100000
Case #15: 13448
Case #16: 8669
Case #17: 37
Case #18: 17
Case #19: 30
Case #20: 19
Case #21: 4238
Case #22: 7337
Case #23: 6262
Case #24: 1850
Case #25: 257
Case #26: 149
Case #27: 163
Case #28: 202
Case #29: 210
Case #30: 3083
Case #31: 969
Case #32: 4718
Case #33: 16885
Case #34: 1589
Case #35: 5989
Case #36: 14012
Case #37: 2489
Case #38: 3559
Case #39: 17745
Case #40: 34350
Case #41: 5426
Case #42: 2368
Case #43: 4693
Case #44: 46032
Case #45: 4618
Case #46: 14620
Case #47: 3848
Case #48: 7197
Case #49: 164
Case #50: 4325
